wallet/src/de/schildbach/wallet/ui/LockScreenActivity.kt

```diff
@@ -211,11 +208,14 @@ open class LockScreenActivity : SecureActivity() {
 
     override fun onStart() {
         super.onStart()
-        setupInitState()
```

Comment:
This doesn't work for the locked state because `USE_DEFAULT` is set immediately after. This also allows unlocking the locked wallet within 10 seconds with a fingerprint.
Moving the locked state check below to fix.